utils/react-native/src/navigation.mjs

```diff
@@ -0,0 +1,114 @@
+import { CommonActions } from '@react-navigation/native'
+
+
+
+export const nav = (() => {
+  class Navigation {
+    set (obj) {
+      this.navigation = obj
+    }
+
+    ///# @name to
+    ///# @description navigate *to* a page
+    ///# https://reactnavigation.org/docs/en/navigation-prop.html#navigate-link-to-other-screens
+    ///# @example
+    ///# nav.to('ProductDetail', { prodId: 13009 })
+    to (...args) {
+      this.navigation.navigate(...args)
+    }
+
+    ///# @name back
+    ///# @description navigate back to a page
+    ///# https://reactnavigation.org/docs/en/navigation-prop.html#goback-close-the-active-screen-and-move-back
+    ///# @arg {null, string} screen [null] - The screen to go back to.
+    ///# If it's null it will just go back to the previous screen
+    ///# @alias `nav.close()`
+    ///# @example
+    ///# // goes back to previous page
+    ///# nav.back()
+    ///# // goes back until it gets to the product detail page
+    ///# nav.back('ProductDetail')
+    back = (screen = null) => this.navigation.goBack(screen)
+
+    close = () => this.back(null)
```

Comment:
Maybe add a comment to indicate this is an alias for `back()`; not necessary but would be consistent with commenting code

Reply:
there is a comment on line 25